### DISCUSSION TITLE
CustomShader는 사용 불가능 한가요?

### DISCUSSION THREAD

**Discussion by Sonak0930** [naverz/zepeto-studio-kor#1162] [5. Q&A]
Unity Asset에서 다운 받은 쉐이더를적용하니까 보이지 않네요
![image](https://user-images.githubusercontent.com/59792911/184272862-d7a01585-683a-4950-ae4a-4f9ac04a89be.png)

**Comment by zlmrzlo** [comment]
제페토 월드에서는 C# 스크립트가 적용되지 않아요.

**Reply by Sonak0930** [reply]
아아 그렇죠 근데 Typescript로 만든 쉐이더는 없는가 궁금해서요.

**Reply by yoon-hw22** [reply]
다른 글 보면 커스텀 쉐이더로 물 구현한 월드가 있는것같은데 쉐이더도 c# 지원불가인가요?

**Reply by zlmrzlo** [reply]
쉐이더는 따로 쉐이더 파일이 존재해요.
쉐이더 파일을 전용으로 구현하시면 될 것 같아요.

**Comment by YSHFM** [comment]
제가 적용한 커스텀 쉐이더 예시입니다

https://www.mettit.com/log/view/LOG20220729131912463

https://www.mettit.com/pt/view/PRE20220512190033673

유니티 쉐이더는 .shader라는 스크립트로 실행됩니다.
C#이나 typescript는 아닙니다.
HLSL기반으로 실행되는 것으로 알고 있습니다.
